src/components/date-picker/__tests__/DatePicker.test.js

```diff
@@ -20,18 +38,6 @@ describe('components/date-picker/DatePicker', () => {
         clock.restore();
     });
 
-    test('should correctly render children in DatePicker', () => {
```

Comment:
Are there any corresponding snapshots to remove?

Reply:
The corresponding snapshots have already been removed.